torch/_dynamo/source.py

```diff
@@ -251,6 +251,30 @@ def name(self):
         return f"type({self.base.name()})"
 
 
+@dataclasses.dataclass
+class SuperSource(Source):
+    type: Source
+    obj: Source
+
+    def __post_init__(self):
+        assert self.type is not None
+        assert self.obj is not None
+
+    def reconstruct(self, codegen):
+        codegen.load_import_from("builtins", "super")
+        return (
+            self.type.reconstruct(codegen)
+            + self.obj.reconstruct(codegen)
+            + [create_instruction("CALL_FUNCTION", 2)]
+        )
+
+    def guard_source(self):
+        return self.obj.guard_source()
+
+    def name(self):
+        return f"super({self.type.name()}, {self.obj.name()})"
```

Comment:
Why can't this just be super(); can super be materially different in Python 3?

Reply:
i think technically you can pass another class as the second obj in the super call, and you'll run MRO but skip some steps.  I don't think this is a common use case, but i figured we might as well capture the super call precisely as it was issued.